vagrant/rtmp-server/bootstrap.sh

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env bash
+
+set -e
+
+YGGDRASIL_GO_COMMIT=b0acc19
+NGINX_VERSION=1.15.0
+
+# Prevent apt-daily from holding up /var/lib/dpkg/lock on boot
+systemctl disable apt-daily.service
+systemctl disable apt-daily.timer
+
+# Install Digital Ocean new metrics
+curl -sSL https://agent.digitalocean.com/install.sh | sh
+
+# Install standard tools
+apt install -y \
+  build-essential \
+  git \
+  libpcre3 \
+  libpcre3-dev \
+  libssl-dev \
+  zlibc \
+  zlib1g \
+  zlib1g-dev
+
+# Install golang
+mkdir /tmp/golang
+wget --progress=bar:force https://dl.google.com/go/go1.9.2.linux-amd64.tar.gz -P /tmp/golang
+tar -C /usr/local -xzf /tmp/golang/go1.9.2.linux-amd64.tar.gz
+{
+  echo ''
+  echo '# Add golang path'
+  echo 'export PATH=$PATH:/usr/local/go/bin'
+} >> /etc/profile
+. /etc/profile
+
+###########
+# OpenVPN #
+###########
+
+# Install openvpn and generate keys and config file
+apt install -y openvpn
+
+# Build certificates
+cd /usr/share/easy-rsa/
+cp openssl-1.0.0.cnf openssl.cnf 
+. ./vars
+./clean-all
+./build-dh
+
+# Run the rest manually to avoid interactivity
+export EASY_RSA="${EASY_RSA:-.}"
+"$EASY_RSA/pkitool" --initca #ca
+"$EASY_RSA/pkitool" --server server #server
+"$EASY_RSA/pkitool" remote #client
+
+# Create server file
+cat <<"EOF"> /etc/openvpn/openvpn.conf
```

Comment:
Could these be separate files like how we have nginx.conf rather than a big cat?